src/Configuration/Config.php

```diff
@@ -49,9 +105,27 @@ public function parseConfig(): Collection
         //'permissions' => $this->parseConfigYaml('permissions.yml'),
         //'extensions' => $this->parseConfigYaml('extensions.yml'),
 
+        $this->setTimestamps($general, $taxonomy, $contentTypes);
+
         return $config;
     }
 
+    private function setTimestamps(...$configs): void
+    {
+        $this->timestamps = [];
```

Comment:
why timestamps are stored both in cache and here?

Reply:
We gather them here, in order to store them in the cache. 

On the next invocation, the values are fetched from the cache, and compared to the current timestamps to determine if the cache is still valid. 
This way, if _any_ of the used config files are modified through any means, the cache is invalidated. The overhead of this is very minimal.

Reply:
but why they are stored in the class property? afair it can be just a variable, because checking timestamps is done only once

Reply:
Ah, yes.. upcoming push will remove it as class property.